### PR TITLE
[DOC] cambio :D

### PR DESCRIPTION
**Pull request by Shadozque**
Pedro y Pablo eran hermanos :)

### CODE REVIEW

File: docs/_posts/2010-01-07-post-standard.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ tags:
 
 All children, except one, grow up. They soon know that they will grow up, and the way Wendy knew was this. One day when she was two years old she was playing in a garden, and she plucked another flower and ran with it to her mother. I suppose she must have looked rather delightful, for Mrs. Darling put her hand to her heart and cried, "Oh, why can't you remain like this for ever!" This was all that passed between them on the subject, but henceforth Wendy knew that she must grow up. You always know after you are two. Two is the beginning of the end.
 
-Mrs. Darling first heard of Peter when she was tidying up her children's minds. It is the nightly custom of every good mother after her children are asleep to rummage in their minds and put things straight for next morning, repacking into their proper places the many articles that have wandered during the day.
+Mrs. Darling first heard of Peter when she was tidying up her children's minds. It is the nightly custom of every good mother after her children are asleep to rummage in their minds and put things straight for next morning, repacking into their proper places the many articles that have wandered during the day. :DD
```
Comment (lemontyc) [inline docs/_posts/2010-01-07-post-standard.md:14]: Please rollback to previous state

**Review by lemontyc — APPROVED** [review]
All good! Magnificent job :D

**Review by carlos-silvap — APPROVED** [review]

**Review by abrahamaldaco — APPROVED** [review]

**Review by LuisAGuzmanI — APPROVED** [review]
Approved

**Review by kmontocam — COMMENTED** [review]
Bien hecho

**Review by mariferdamian — COMMENTED** [review]
good

**Comment by mariferdamian** [comment]
hoola

**Pull request merged**